content/en/docs/project-user-guide/application-workloads/statefulsets.md

```diff
@@ -21,18 +19,21 @@ StatefulSets are valuable for applications that require one or more of the follo
 - Ordered, graceful deployment, and scaling.
 - Ordered, automated rolling updates.
 
+For more information, see the [official documentation of Kubernetes](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/).
+
 ## Prerequisites
 
 - You need to create a workspace, a project and an account (`project-regular`). Please refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project) if they are not ready yet.
-- You need to sign in with `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+- You need to sign in with `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [the steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
 
 ## Create a StatefulSet
 
-In KubeSphere, a **Headless** service is also created when you create a StatefulSet. You can find the headless service in **Services** under **Application Workloads** in a project.
+In KubeSphere, a **Headless** service is also created when you create a StatefulSet. You can find the headless service in [Services](../services/) under **Application Workloads** in a project.
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **StatefulSets** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **StatefulSets** and click **Create**.
```

Comment:
Same here. Please change it to:

Go to **Application Workloads** of a project, select **Workloads**, and click **Create** under the tab **StatefulSets**.